### ISSUE TITLE
Feature Request: "unblock YouTube" button

### ISSUE THREAD

**Issue by pjlegato** [gorhill/uMatrix#133]
Many sites use embedded YouTube videos. Sometimes, I want to unblock them, but this requires several page reloads as each unblocked element progressively loads additional elements from other domains associated with YouTube (ytimg.com, google.com, googlevideo.com, sometimes others.) Setting them all to 'allowed' typically requires unblocking them one by one and reloading the entire page 3-4 times in between, to get the next element to load..

Since "unblocking embedded YouTube" videos is a common operation, it'd be nice to have a facility for instructing uMatrix to "unblock all elements necessary to play embedded YouTube videos" as a unit.

**Comment by RandomAcronym** [comment]
Support for macros in general sounds interesting to me.

**Comment by gorhill** [comment]
Duplicate of #30.
